tests/layer_tests/tensorflow_lite_tests/test_tfl_BinaryWithActivation.py

```diff
@@ -0,0 +1,76 @@
+import itertools
+
+import tensorflow as tf
+import pytest
+
+from common.tflite_layer_test_class import TFLiteLayerTest
+from tensorflow_lite_tests.test_tfl_Unary import data_generators
+
+test_ops = [
+    {'op_name': 'ADD', 'op_func': tf.math.add},
+    {'op_name': 'DIV', 'op_func': tf.math.divide, 'kwargs_to_prepare_input': 'positive'},
+    {'op_name': 'MUL', 'op_func': tf.math.multiply},
+    {'op_name': 'SUB', 'op_func': tf.math.subtract},
+]
+
+test_params = [
+    {'shape': [2, 10, 10, 3]},
+    {'shape': [2, 10]}
+]
+
+activations = [
+    {'activation': None},
+    {'activation': tf.nn.relu},
+    {'activation': tf.nn.relu6},
+    {'activation': tf.math.tanh},
+    {'activation': tf.experimental.numpy.signbit},
+    # n1
```

Comment:
will you cover the relu n1 case?
Additionally, fused activations are shared between several operations -- you can move this list and activation_helper to some utils.py

Reply:
"will you cover the relu n1 case?" - Yes
"you can move this list and activation_helper to some utils.py" - I will move all such functions in some utils.py